text/0000-customisability.md

```diff
@@ -0,0 +1,86 @@
+- Start Date: 2016-10-25
+- RFC Issue: #3
+- RFC PR: 
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Customisation of Rustfmt should be allowed (via a `rustfmt.toml` file), but not
```

Comment:
Not that my opinion matters much here but I agree regarding `rustfmt`. The tool (and repo) is named `rustfmt` so that solidifies the name to me.